tests/scripts/thread-cert/Cert_6_1_03_RouterAttachConnectivity.py

```diff
@@ -91,10 +98,42 @@ def test(self):
         self.simulator.go(5)
         self.assertEqual(self.nodes[ED].get_state(), 'child')
 
+        self.collect_ipaddrs()
         addrs = self.nodes[ED].get_addrs()
         for addr in addrs:
             self.assertTrue(self.nodes[ROUTER3].ping(addr))
 
+    def verify(self, pv):
+        pkts = pv.pkts
+        pv.summary.show()
+
+        ROUTER_3 = pv.vars['ROUTER_3']
+        ED = pv.vars['ED']
+        _router3_pkts = pkts.filter_wpan_src64(ROUTER_3)
+        _ed_pkts = pkts.filter_wpan_src64(ED)
+
+        # Step 2: The DUT MUST send a MLE Parent Request to the
+        # All-Routers multicast address
```

Comment:
Are we validating the all-routers multicast address?

Reply:
yes, we should. Added in the commit 6